common/djangoapps/student/management/tests/test_bulk_unenroll.py

```diff
@@ -0,0 +1,103 @@
+from tempfile import NamedTemporaryFile
+
+from django.core.management import call_command
+from xmodule.modulestore.tests.django_utils import SharedModuleStoreTestCase
+from xmodule.modulestore.tests.factories import CourseFactory
+from testfixtures import LogCapture
+
+from course_modes.tests.factories import CourseModeFactory
+from student.tests.factories import UserFactory
+from student.models import CourseEnrollment, User
+
+
+LOGGER_NAME = 'student.management.commands.bulk_unenroll'
+
+
+class BulkUnenrollTests(SharedModuleStoreTestCase):
+    """Test Bulk un-enroll command works fine for all test cases."""
+    def setUp(self):
+        super(BulkUnenrollTests, self).setUp()
+        self.course = CourseFactory.create()
```

Comment:
Can you add a unicode character into the course name here to test unicode?

Reply:
@robrap can you refer me to an example where we are doing this?

Reply:
I just did some searches...

Here is an example of setting courserun details:
https://github.com/edx/edx-platform/blob/c125ff6a57ad09171058191a8482184644fd9b8b/openedx/core/djangoapps/credit/tests/test_api.py#L161
Here is an example of unicode data:
https://github.com/edx/edx-platform/blob/1fd2167144425a24f866b63da0e2fd1b9454c34f/common/lib/xmodule/xmodule/tests/test_lti20_unit.py#L177
In order for the unicode to not break the file, that test file needs to be marked appropriately for utf-8: https://github.com/edx/edx-platform/blob/1fd2167144425a24f866b63da0e2fd1b9454c34f/common/lib/xmodule/xmodule/tests/test_lti20_unit.py#L1